src/seedu/addressbook/AddressBook.java

```diff
@@ -480,16 +480,31 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<HashMap<Per
      */
     private static ArrayList<HashMap<PersonProperty, String>> getPersonsWithNameContainingAnyKeyword(Collection<String> keywords) {
         final ArrayList<HashMap<PersonProperty, String>> matchedPersons = new ArrayList<>();
+        Collection<String> lowerCaseKeywords = setStringsToLowerCase(keywords);
         for (HashMap<PersonProperty, String> person : getAllPersonsInAddressBook()) {
             final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            if (!Collections.disjoint(setStringsToLowerCase(wordsInName), lowerCaseKeywords)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
 
     /**
+     * Takes as input a collection of Strings and returns a new collection with the Strings in lowercase.
```

Comment:
The header comment and the descriptions in @param and @ return are redundant right? Can you improve the header comment?